cmake/kokkos_arch.cmake

```diff
@@ -378,6 +378,17 @@ IF (KOKKOS_ARCH_HSW)
   )
 ENDIF()
 
+IF (KOKKOS_ARCH_RISC64_rv64imafdcv)
```

Comment:
I do not like the prefix `KOKKOS_ARCH_RISC64`, I think `KOKKOS_ARCH_RISCV` is better considering that other flavors of risc exist out there that are unrelated to RISC-V.

Reply:
Ok, I can change that.